### PR TITLE
chore: add configurations to run on github actions workflows

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER-9234](https://2u-internal.atlassian.net/browse/LEARNER-9234)

**Description**

- Add required capabilities to run on Github actions.
- Add emulator and OS version details of Github actions in test repo so that the emulator can run automation test cases on it.
- Make configurations separate from running the test cases locally.

### CODE REVIEW

File: tests/common/globals.py [hunk 1]
```diff
@@ -95,18 +96,23 @@ def setup_global_environment(self):
...
+            self.android_platform_version = user_preferences.get('Settings').get('android_platform_version')
+            self.ios_platform_version = user_preferences.get('Settings').get('ios_platform_version')
+            self.aut_current_path = user_preferences.get('Settings').get('aut_current_path')
+            self.aut_latest_path = user_preferences.get('Settings').get('aut_latest_path')
+            self.login_user_name = user_preferences.get('User').get('login_user_name')
+            self.login_password = user_preferences.get('User').get('login_password')
+
+        else:
+            self.server_url = 'http://127.0.0.1:4723/wd/hub'
+            self.target_environment = 'Android'
```
Comment (moeez96) [inline tests/common/globals.py:115]: Can we have these constants declared as variables in a file?

File: tests/conftest.py [hunk 2]
```diff
@@ -36,16 +36,27 @@ def set_capabilities(setup_logging):
...
+
+    if globals_contents.enable_workflows is True:
+        desired_capabilities['appWaitDuration'] = '50000'
+        desired_capabilities['platformName'] = strings.ANDROID
+        desired_capabilities['platformVersion'] = '10'
+        desired_capabilities['deviceName'] = 'emulator-5554'
+        desired_capabilities['appPackage'] = globals_contents.AUT_PACKAGE_NAME
+        desired_capabilities['appActivity'] = Globals.SPLASH_ACTIVITY_NAME
+        desired_capabilities['appWaitActivity'] = Globals.NEW_LOGISTRATION_ACTIVITY_NAME
+        desired_capabilities['automationName'] = 'UiAutomator2'
```
Comment (moeez96) [inline tests/conftest.py:57]: Ideal to have constant values declared as variables ina file/module and then used by importing.

**Review by moeez96 — APPROVED** [review]
Approved with few nits. LGTM

File: tests/common/globals.py (hunk 1 above)
Comment (bilalawan321) [reply tests/common/globals.py:115]: fixed

**Pull request merged**